src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -193,14 +193,15 @@ Scalar<DataVector> spin_function(
     const tnsr::I<DataVector, 3, Frame>& unit_normal_vector,
     const Scalar<DataVector>& area_element,
     const tnsr::ii<DataVector, 3, Frame>& extrinsic_curvature) noexcept {
-  auto sin_theta = make_with_value<Scalar<DataVector>>(area_element, 0.0);
+  DataVector temp(get(area_element).size(), 0.0);
```

Comment:
oops sorry, the temp should be a Scalar<DataVector> to be able to use as the return value.
so `auto temp = make_with_value<Scalar<DataVector>>(area_element, 0.0);`